lib/Site.js

```diff
@@ -99,7 +99,7 @@ const GENERATE_SITE_LOGGING_KEY = 'Generate Site';
 const MARKBIND_WEBSITE_URL = 'https://markbind.github.io/markbind/';
 const MARKBIND_LINK_HTML = `<a href='${MARKBIND_WEBSITE_URL}'>MarkBind ${CLI_VERSION}</a>`;
 
-function Site(rootPath, outputPath, forceReload = false) {
+function Site(rootPath, outputPath, forceReload = false, siteConfigPath = SITE_CONFIG_NAME) {
   const configPath = findUp.sync(SITE_CONFIG_NAME, { cwd: rootPath });
```

Comment:
Change `SITE_CONFIG_NAME` to `siteConfigPath`, so that user do not have to necessary serve it at the root.

Reply:
Hi, you mean allow siteConfigPath to be a filepath (like `configs/mysite.json`)? I think if we add that functionality then `markbind serve` must be run in the root folder (not subdirectory), since markbind currently uses the site.json folder to identify the root folder.

Alternatively we can have a different method of locating the root folder.

Reply:
No, I mean that it be changed so that it becomes from:

```js
const configPath = findUp.sync(SITE_CONFIG_NAME, { cwd: rootPath });
```

to 

```js
const configPath = findUp.sync(siteConfigPath, { cwd: rootPath });
```

Making this change still allows the user to serve the site inside subdirectories, unless the directory of `mysite.json` is different from `site.json` (then again in such a case the directory where `mysite.json` should take precedent as the root directory). `siteConfigPath` need not be a filepath for it to work (it can be just `mysite.json`).

Reply:
Ah, gotcha